DailySelfie/app/src/main/java/com/teamagam/dailyselfie/LoadPictureTask.java

```diff
@@ -10,19 +10,23 @@
 import java.lang.ref.WeakReference;
 
 class LoadPictureTask extends AsyncTask<String, Void, Bitmap> {
-    private final WeakReference<ImageView> mImageViewReference;
-    private final int mImageSize;
 
-    LoadPictureTask(ImageView imageView, int imageSize) {
+    protected final WeakReference<ImageView> mImageViewReference;
+
+    public LoadPictureTask() {
+        super();
+        mImageViewReference = null;
+    }
+
+    LoadPictureTask(ImageView imageView) {
         // Use a WeakReference to ensure the ImageView can be garbage collected
         mImageViewReference = new WeakReference<>(imageView);
-        mImageSize = imageSize;
     }
 
     @Override
     protected Bitmap doInBackground(String... params) {
         String path = params[0];
-        return decodeSampledBitmapFromFile(path, mImageSize, mImageSize);
+        return BitmapFactory.decodeFile(path);
     }
 
     @Override
```

Comment:
I cannot comment on line 36, but it seems like you're checking the weak reference for null value twice.